aiodocker/jsonstream.py

```diff
@@ -42,7 +42,7 @@ def __aiter__(self):
         # (see https://github.com/KeepSafe/aiohttp/issues/739)
 
         # response error , it has been closed
-        await self._response.close()
+        self._response.close()
```

Comment:
No I actually fixed it. This is no coroutine apparently and the try...except...pass on the top level was hiding the exception. Commit message: 1d3abd8c570eb66e2fcc9c0b297c9d2a9153c7a3